tsconfig.json

```diff
@@ -1,7 +1,28 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
 {
-  "extends": "./node_modules/@skyux-sdk/builder/tsconfig",
+  "compileOnSave": false,
   "compilerOptions": {
-    "module": "commonjs"
+    "baseUrl": "./",
+    "outDir": "./dist/out-tsc",
+    "sourceMap": true,
+    "declaration": false,
+    "downlevelIteration": true,
+    "experimentalDecorators": true,
+    "moduleResolution": "node",
+    "importHelpers": true,
+    "target": "es2017",
+    "paths": {
+      "@skyux/pages": [
```

Comment:
The `paths` should include what Angular generates automatically, plus the override for testing modules:
```
  "paths": {
      "pages": [
        "dist/pages/pages",
        "dist/pages"
      ],
      "@skyux/pages": [
        "projects/pages/src/public-api.ts"
      ]
    },
```